web/modules/custom/geolayer/modules/map/geolayer_map.libraries.yml

```diff
@@ -0,0 +1,35 @@
+geolayer-map:
+  js:
+    /libraries/custom/geolayer-map/dist/geolayer-map.js:
```

Comment:
This library is not in the branch. Also as I mentioned in another comment here let's give it a more generic name, nfa_map. 
I think we should fork https://github.com/farmOS/farmOS-map to National-Forestry-Authority/nfa_map and include it in the project in composer.json. It would look something like this:
```
 {
        "type": "package",
        "package": {
            "name": "National-Forestry-Authority/nfa_map",
            "version": "main",
            "type": "drupal-library",
            "source": {
                "url": "https://github.com/National-Forestry-Authority/nfa_map",
                "type": "git",
                "reference":"main"
            }
        }
}
```

Reply:
@peacog apologies my git was acting up , it was not taking the changes in the `/libraries/custom/geolayer-map` path for some reason, i deleted the module and added it back and it started detecting it, however when i pushed the module and the changes went something in lakhs for this PR, which i guess is not good to have these many changes in a single PR,so i reverted those changes for now, I like the idea of creating a fork and adding the library as dependency in `composer.json` , but i guess i don't have the permissions to do that, so please do lmk how we can tackle this one .

moreover since the `farmOS-map` module is being used in many different places as a library only for now, do you think it would be better to create a separate task for this ? and update the module in all repos as a part of this task ?

the reason why i am asking this is, you mentioned you needed this module for the `Forest reserve` changes you are working on, and this is a working PR and we can merge this after some minor refactoring , so i was wondering if we should tackle this in a different PR and add the map library as a part of libraries folder only for now. 

please lmk your thoughts on these and i can update the PR accordingly !

Reply:
That's a good suggestion @Yash621 . I'll create a new ticket for the task of create a fork of farmOS-map.

There's no need to add the custom geolayer-map library to this PR. I have squashed and merged your branch into 47-geolayer-entity and I have included in composer.json there the farmOS-map library. You can continue working on the 47-geolayer-entity branch https://github.com/National-Forestry-Authority/brms/pull/59